apps/web/src/pages/integrations/components/multi-provider/SelectProviderSidebar.tsx

```diff
@@ -99,8 +99,10 @@ export function SelectProviderSidebar({
   };
 
   useEffect(() => {
-    onTabChange(scrollTo?.toString());
-  }, [onTabChange, scrollTo]);
+    setTimeout(() => {
+      onTabChange(scrollTo?.toString());
+    }, 100);
+  }, []);
```

Comment:
there will be an ESLint warning about missing deps, please fix it

Reply:
Sure @LetItRock ,Just added the dependency . Was just curios about why we are putting onTabChange function as deps , we are scrolling once we land to the page .After that any tab changes , the onTabChange function will trigger the scrolling .

Reply:
it's safe to put the `onTabChange` here in the deps array, because it's memoized and won't change during the re-renders ;)

Reply:
Got it 👍